### ISSUE TITLE
Please add support for "selective" reruns for jest triggered from Allure TestOps screen

### ISSUE THREAD

**Issue by balazsgalambos** [allure-framework/allure-js#442]
**Is your feature request related to a problem? Please describe.**
I'd like the allure jest plugin to be handle rerun trigger a rerun of the selected tests on AllureTestOPS screen.

**Describe the solution you'd like**
I'd like the allure-jest plugin can 
* integrate well with CI plugin (jenkins)
*  digest allureEE/testplan.json file
* configure jest test run to filter/run tests based on this selection only.

**Describe alternatives you've considered**
Any other way I can integrate selective reruns with Jest framework.

**Additional context**
Currently rerunning from allureEE and Jest based projects triggers a full rerun of the job.
We have come up with our own implementation of handling the selection and configuring jest to only rerun the selected ones.
We are also a bit worried if an incompatible change is to happen in allure side (eg: the format or way of getting testplan.json) we will loose rerun ability for our JS/TS tests, and will need to spend much effort reimplementing it.

**Comment by gicoprasico** [comment]
Yes please, this would be an awesome feature.

**Comment by ryust-ax** [comment]
I don't have experience with jest as a primary runner.  However, in exploring TestOps while using CodeceptJS, I ran into the same difficulty (unable to selectively execute).  I do have a modification implemented in my configuration file bootstrap which accomplishes what I want.  Here's the link to my post in CodeceptJS project describing what I found in the dependencies and then my bootstrapping implementation.  Maybe it'll help as a work-around or as inspiration for a project update.
https://github.com/codeceptjs/CodeceptJS/issues/3544
